package.json

```diff
@@ -12,7 +12,8 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "tsc --allowSyntheticDefaultImports",
```

Comment:
Can you specify why the `--allowSyntheticDefaultImports` flag is needed here and not in `build:js`